drafts/draft-ietf-ippm-ioam-data-integrity.xml

```diff
@@ -912,18 +894,59 @@
       protection of IOAM-Data-Fields. The details of the key exchange are
       outside the scope of this document.</t>
 
-      <t>The Integrity Protection Method defined in this document requires
-      additional per-packet processing by each node that uses it. Inappropriate
-      use of the Integrity Protection Method might overload nodes and cause them
-      to stop functioning properly. Operators deploying IOAM with the Integrity
-      Protection Method MUST ensure that such overload situations are avoided.
-      This could for example be achieved by applying IOAM only to a subset of
-      the entire traffic.</t>
-
-      <t>The Nonce makes a signature chain unique but does not necessarily
-      prevent replay attacks. To enable replay protection, the encapsulating node
-      and the Validator(s) MUST agree on a common methodology to keep the Nonce valid
-      only for a specific period of time, which is outside the scope of this document.</t>
+      <t>There is an additional per-packet processing for each node that uses
+      the Integrity Protection Method defined in this document. Inappropriate
+      use of this Integrity Protection Method might overload nodes and cause
+      them to stop functioning properly. Operators deploying IOAM with this
+      Integrity Protection Method MUST ensure that such overload situations are
+      avoided. This could for example be achieved by applying IOAM only to a
+      subset of the entire traffic.</t>
```

Comment:
Mention the obvious caveat: If you don't protect all packets with IOAM, the unprotected packets are unprotected.